app/views/profiles/show.html.erb

```diff
@@ -73,6 +73,18 @@
           </p>
         <% end %>
 
+        <% if @user.mastodon_handle.present? %>
+          <%=
+            image_tag(
+              "/images/mastodon_icon.png",
+              alt: "Mastodon icon",
+              class: "profile__header__icon"
+            )
+          %>
+
+          <%= mastodon_handle(user) %>
```

Comment:
Isn't it possible to generate link using mastodon handle? :thinking:

Reply:
@simi I wasn't able to find a way to do this, it's also weird if we just let anyone put any link they want on the profile? We probably have to sanitize it, etc.

Reply:
Isn't enough to just split on last `@` and reverse join with `/` (and prepending `https` protocol)?